tests/unit/azure/test_locations.py

```diff
@@ -266,6 +266,110 @@ def test_run_managed_identity():
     )
 
 
+def test_run_access_connectors():
+    """Test run with access connectors based storage credentials"""
+    mock_backend = MockBackend(
+        rows={
+            r"SELECT \* FROM location_test.external_locations": MockBackend.rows("location", "table_count")[
```

Comment:
`MockBackend.rows("location", "table_count")` is repeated quite a few times, we can make it a constant, similar to `test_grants.py`